c/datasetService.c

```diff
@@ -99,13 +99,16 @@ static int serveDatasetContents(HttpService *service, HttpResponse *response){
     respondWithDataset(response, filename, TRUE);
   }
   else if (!strcmp(request->method, methodPOST)){
+    /* this section is WIP to allow saving of members when another user
```

Comment:
Should this be part of the pull-request if it's WIP?

Reply:
I don't think that comment came from me.  That ELSE block AT LINE 101 contains the same code as `staging`, apart from the comment, so I guess someone else removed the comment.  I did add the EOL comments for my own interest on lines 106+107, but I made no other change to that ELSE block

Reply:
> I don't think that comment came from me. 

Al the commits in this pull-request belong to you. I don't see who else could have added this comment.

Reply:
Yes, you're right, the comment is mine.  Now I remember.  I added it as a placeholder to remind me that there was a problem to be fixed.  So the comment refers to code I've not written yet.

The problem is, if a user has the PDS directory open, the POST code at line 101 won't work.  This is because the POST accesses the dataset via USS.  If you try it yourself at the USS command prompt, you will find that writing to a member of that PDS won't work while the directory is open.

I have a solution for this, but I've not had time to finalise it.

Reply:
hopefully removed extraneous comments